core/src/main/java/jenkins/model/item_category/NestedProjectsCategory.java

```diff
@@ -0,0 +1,38 @@
+package jenkins.model.item_category;
+
+import hudson.Extension;
+import jenkins.model.Messages;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+
+/**
+ * A generic {@link ItemCategory}.
+ *
+ * This category should be moved to cloudbees-folder-plugin short-term.
```

Comment:
Then why is it here - this javadoc entry makes no sense to me.

missing `@since 2.0`?